portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,291 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
+    <meta charset="UTF-8" />
+    <title>Home - James Tan</title>
+    <link rel="stylesheet" href="style.css" />
     <script src="script.js"></script>
```

Comment:
To improve the website performance better, you can read an article below
https://flaviocopes.com/javascript-async-defer/

Reply:
> To improve the website performance better, you can read an article below
> https://flaviocopes.com/javascript-async-defer/

Noted, I'll be adding defer along with some of the minor tweaks to improve the ergonomics of the website..